### DISCUSSION TITLE
How to go about adding a custom button adjacent to the Clone and Delete buttons for a field.

### DISCUSSION THREAD

**Discussion by Tomdanizer** [vueform/builder#73] [Questions]
I'm wondering how I would go about ideally creating a custom option/button that would appear next to the Clone & Delete buttons. Alternatively utilizing a slot on each element at a global level would also work. I've toyed around with the plugin concept but am stuck. I think templates would work as well but would like to avoid having to create one for every component.

**Comment by adamberecz** [comment]
Unfortunately this is not something you can do in the builder at the moment. I've created a ticket for it though: https://github.com/vueform/builder/discussions/74
